aiobotocore/endpoint.py

```diff
@@ -44,9 +48,45 @@ def __getattr__(self, item):
 
     def __dir__(self):
         attrs = dir(self.__content)
-        attrs.append('close')
+        attrs += ['read_all', 'text', 'json' 'close']
         return attrs
 
+    def __del__(self):
+        self.close()
+
+    @asyncio.coroutine
+    def read_all(self):
+        """
```

Comment:
can we hold on on adding this changes? I want to get red of  `__getattr__` all together and do right thing without magic at all.